sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -465,6 +446,7 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
       silentRequest.resourceRequestMethod = options.proofOfPossessionOptions.resourceRequestMethod;
       silentRequest.resourceRequestUri = options.proofOfPossessionOptions.resourceRequestUrl;
     }
+    await app.getTokenCache().getAllAccounts();
```

Comment:
Apparently I need to add this because of a bug in msal -https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/7303#issuecomment-2568789709 
@xirzec

Reply:
With this fix in msal-node https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/9ae4c0ef2926e389f3637c260ea4d4de3e294517 I no longer need this statement.

Reply:
Fixed